src/status_im/common/signals/events.cljs

```diff
@@ -33,61 +38,75 @@
   ;; won't impact performance, as it is a fairly costly operation on large-ish
   ;; data structures
   (let [^js data     (.parse js/JSON event-str)
-        ^js event-js (.-event data)
-        type         (.-type data)]
-    (log/debug "Signal received" event-str)
+        ^js event-js (oops/oget data :event)
```

Comment:
why do we use oops here?

Reply:
is this safe to use here? it throws an error if it's nil :/

Reply:
@J-Son89, that's a good question. 

`oops` should pretty much always be used due to advanced compilation. The checks it perform, according to its documentation only happen during development. In advanced compilation, all checks should disappear.

https://github.com/binaryage/cljs-oops?tab=readme-ov-file#play-it-safe-during-development

Reply:
Can't remember from my experience if this holds true. I'm sure we'll find out soon if not

Reply:
> Can't remember from my experience if this holds true. I'm sure we'll find out soon if not

I'm not sure either. I'll do a check today to settle this question and will report here.

Reply:
@J-Son89, I just confirmed in a prod build that `oops/oget` does not throw any exception on missing keys, as is documented in the library.

To give a real example, this is a warning produced now when we do advanced compilation (release build):

```
------ WARNING #165 - :infer-warning -------------------------------------------
 File: /build/status-mobile-source-jsbundle/src/status_im/contexts/wallet/wallet_connect/effects.cljs:46:10
--------------------------------------------------------------------------------
  43 |    (let [{:keys [params id]} proposal
  44 |          approved-namespaces (wallet-connect/build-approved-namespaces params
  45 |                                                                        supported-namespaces)]
  46 |      (-> (.approveSession web3-wallet
----------------^---------------------------------------------------------------
 Cannot infer target type in expression (. web3-wallet approveSession (clj->js {:id id, :namespaces approved-namespaces}))
```

This error disappears with the correct type hint or when we use `oops/oget`.

But that's not the main problem. Advanced compilation renames properties, unless the code access object properties as strings, but we don't do that when we write `(.x someObject)`. `oops` solves this important problem too.

For the sake of visibility to more folks to help spread the word, cc'ing @clauxx and @smohamedjavid